src/core/Geometry.js

```diff
@@ -232,7 +232,9 @@ Geometry.prototype = Object.assign( Object.create( EventDispatcher.prototype ),
 		var tempUVs = [];
 		var tempUVs2 = [];
 
-		for ( var i = 0, j = 0; i < positions.length; i += 3, j += 2 ) {
+		var colorStride = colors !== undefined ? attributes.color.itemSize : 3;
```

Comment:
If `colors` is undefined, then setting `colorStride` to 3 is not meaningful. Oh well...